python/tk_multi_publish2/dialog.py

```diff
@@ -235,8 +246,31 @@ def __init__(self, parent=None):
         self._display_action_name = self._bundle.get_setting("display_action_name")
         self.ui.publish.setText(self._display_action_name)
 
+        # UI tweaks based on the 'manual_load_enabled' property
+        # disabling drag & drop and browsing for files or folders.
+        #
+        # hide the drop area ( dashed-line border + plus icon )
+        self.ui.large_drop_area.setVisible(self.manual_load_enabled)
+        self.ui.large_drop_area_label.setVisible(self.manual_load_enabled)
+        self.ui.label.setVisible(self.manual_load_enabled)
+        self.ui.drop_area_browse_file.setVisible(self.manual_load_enabled)
+        self.ui.drop_area_browse_seq.setVisible(self.manual_load_enabled)
+        self.ui.browse.setVisible(self.manual_load_enabled)
+        # The tiny label at bottom of the tree view
+        self.ui.text_below_item_tree.setVisible(self.manual_load_enabled)
+        # Override tree view setting based on application 'enable_manual_load' option.
+        # This is mostly about preventing OS visual indicators that user can drop
+        # something on the target.
+        self.ui.items_tree.setDragEnabled(self.manual_load_enabled)
```

Comment:
as mentioned before, i don't think we can do this. the tree view allows internal drag/drop which is independent of the manual load. I think the disabling of external file drag/drop has to be deeper in the code, likely in the `on_drop` slot.